3-enrich/spark-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobConfig.scala

```diff
@@ -145,9 +145,15 @@ object EnrichJobConfig {
    * @param registry EnrichmentRegistry used to find the files that need caching
    * @return A list of URIs representing the files that need caching
    */
-  private def filesToCache(registry: EnrichmentRegistry): List[(URI, String)] =
-    registry.getIpLookupsEnrichment match {
+  private def filesToCache(registry: EnrichmentRegistry): List[(URI, String)] = {
+    val ipLookupFiles = registry.getIpLookupsEnrichment match {
       case Some(ipLookups) => ipLookups.dbsToCache
       case None => Nil
     }
+    val iabFiles = registry.getIabEnrichment match {
+      case Some(iabEnrichment) => iabEnrichment.dbsToCache
+      case None => Nil
+    }
+    ipLookupFiles ++ iabFiles
+  }
```

Comment:
I think this should be defined in the enrichment registry, I logged #3789. Of course, it's out of scope for your release.